tools/GraphClust/LocARNAGraphClust/locarna_best_subtree.xml

```diff
@@ -160,23 +221,23 @@ This saves the calculation of pairwise all-vs-all similarities and construction
                   characterized.}
 }
 
-					</citation>
-					<citation type="bibtex">@Article{Smith:Heyne:Richter:Freib_RNA_Tools:NAR2010,
-  author =	 {Smith, Cameron and Heyne, Steffen and Richter, Andreas S.
+          </citation>
+          <citation type="bibtex">@Article{Smith:Heyne:Richter:Freib_RNA_Tools:NAR2010,
```

Comment:
True, it would have been easier. For future citations we take care of using doi.